src/OSPSuite.Infrastructure.Import/Core/DataSource.cs

```diff
@@ -16,15 +19,15 @@ public interface IDataSource
    {
       void SetDataFormat(IDataFormat dataFormat);
       void SetNamingConvention(string namingConvention);
-      void AddSheets(Cache<string, DataSheet> dataSheets, IReadOnlyList<ColumnInfo> columnInfos, string filter);
+      Cache<IDataSet, List<ParseErrorDescription>> AddSheets(Cache<string, DataSheet> dataSheets, IReadOnlyList<ColumnInfo> columnInfos, string filter);
```

Comment:
This is an unexpected signature. What does this returned cache represent? .Can you comment the interface so that we know what's going on?
In general, returning a cache is not commun weird because you are exposing the API to the consumer but it might be required